src/pyscipopt/scip.pxi

```diff
@@ -4227,14 +6953,25 @@ cdef class Model:
         Py_INCREF(cutsel)
 
     def includeBranchrule(self, Branchrule branchrule, name, desc, priority, maxdepth, maxbounddist):
-        """Include a branching rule.
+        """
+        Include a branching rule.
 
-        :param Branchrule branchrule: branching rule
-        :param name: name of branching rule
-        :param desc: description of branching rule
-        :param priority: priority of branching rule
-        :param maxdepth: maximal depth level up to which this branching rule should be used (or -1)
-        :param maxbounddist: maximal relative distance from current node's dual bound to primal bound compared to best node's dual bound for applying branching rule (0.0: only on current best node, 1.0: on all nodes)
+        Parameters
+        ----------
+        branchrule : Branchrule
+            branching rule
+        name : str
+            name of branching rule
+        desc : str
+            description of branching rule
+        priority : int
+        priority of branching rule
```

Comment:
I can't tab in here, but there's a tab missing

Reply:
Fixed!